vocabularies/PDF.xml

```diff
@@ -115,6 +115,12 @@
             <String>PDF supports setting the text direction-layout (e.g. left-to-right or right-to-left) in the document description</String>
           </Annotation>
         </Property>
+        <Property Name="UploadToFileShare" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
+          <Annotation Term="Core.Description" String="Support of PDF document upload"/>
+          <Annotation Term="Core.LongDescription">
+            <String>Is the upload of a PDF document to a file share server supported?</String>
```

Comment:
Why "to a file share server"? Does the client need to know that?

Reply:
Yes, the upload to a "File Share" is the specific feature that is advertised here. See also the adapted 
LongDescription.